climada/entity/exposures/crop_production.py

```diff
@@ -27,9 +27,10 @@
 import h5py
 from matplotlib import pyplot as plt
 from iso3166 import countries as iso_cntry
+
 from climada.entity.exposures.base import Exposures
 from climada.entity.tag import Tag
-import climada.util.coordinates as coord
+import climada.util.coordinates as co
```

Comment:
Since it also affected `trop_cyclone.py`, I used this PR to unify the shorthand for `climada.util.coordinates` everywhere in the whole project. The shorthand `co` is taken from the base modules in the hazard and exposures subpackages.

Reply:
I would suggest to go the other way around. 'co' is very undescriptive. Since it is not a very often used module such as numpy or  pandas, this would improve readability. What do you think?